2/contrib/jenkins/install-plugins.sh

```diff
@@ -172,9 +173,8 @@ function download() {
     plugin="$1"
     version="${2:-latest}"
     ignoreLockFile="${3:-}"
-    lock="$(getLockFile "$plugin")"
 
-    if [[ $ignoreLockFile ]] || mkdir "$lock" &>/dev/null; then
+    if [[ $ignoreLockFile ]] || ! test -f $(getLockFile $plugin); then
```

Comment:
is there any reason to change the lockfile from a directory to a file?
I don't know the historical reason of having a directory, but, if we can minimize changes to this old code, that would be more reinsuring.

Reply:
I am storing the locked plugin version in the `<plugin>.lock` file for use when checking the versions that are being requested.